### PR TITLE
Feature/password expiration

### PR DESCRIPTION
**Pull request by Nainterceptor**
Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

File: components/form/password.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<template>
+  <div>
+    <input
+      v-bind="$attrs"
```
Comment (Scttpr) [inline components/form/password.vue]: I thought it was better to user v-model ?

File: components/form/password.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      type="password"
+      class="input"
```
Comment (Scttpr) [inline components/form/password.vue]: Is it compulsory ?

File: components/form/password.vue (hunk 1 above)
Comment (Scttpr) [reply components/form/password.vue]: Ok just saw that you pass v-model from parent ! :)

File: pages/login.vue [hunk 3]
```diff
@@ -68,9 +68,27 @@ export default {
...
+          case 404:
+            this.$toast.error('Impossible de se connecter, l\'utilisateur n\'existe pas. '
+              + 'Merci de vérifier votre identifiant.\n'
+              + 'Si le problème persiste, contactez le régulateur pour confirmer votre identifiant.');
+            break;
+          case 403:
+            this.$toast.error('Impossible de se connecter, le mot de passe est incorrect. '
+              + 'Merci de retaper votre mot de passe.\n'
+              + 'Si le problème persiste, contactez le régulateur pour réinitialiser votre mot de passe.');
+            this.user.password = null;
```
Comment (Scttpr) [inline pages/login.vue]: Default value is empty string, is it better to use null here ?

File: pages/login.vue [hunk 4]
```diff
@@ -68,9 +68,27 @@ export default {
...
+            break;
+          case 401:
+            this.$toast.error('Impossible de se connecter, le mot de passe a expiré. '
+              + 'Contactez le régulateur pour le réinitialiser.');
```
Comment (Scttpr) [inline pages/login.vue]: I would clean up input value here as well

File: pages/login.vue [hunk 5]
```diff
@@ -68,9 +68,27 @@ export default {
...
+            break;
+          default:
+            this.$toast.error('Une erreur est survenue, merci de vérifier votre email et mot de passe. '
+              + 'Contactez le régulateur pour réinitialiser le mot de passe.');
```
Comment (Scttpr) [inline pages/login.vue]: Same about input value cleanup

File: components/form/password.vue (hunk 2 above)
Comment (Nainterceptor) [reply components/form/password.vue]: the class ? yes, it is bulma class 🤷‍♂️

File: pages/login.vue (hunk 4 above)
Comment (Nainterceptor) [reply pages/login.vue]: We're cleaning password when the password is wrong, it's not the case here.

File: pages/login.vue (hunk 5 above)
Comment (Nainterceptor) [reply pages/login.vue]: We're cleaning password when the password is wrong, it's not the case here.

File: pages/login.vue (hunk 3 above)
Comment (Nainterceptor) [reply pages/login.vue]: Mhhh, it's working with null, code has been copied from login.vue of dashboard

File: components/form/password.vue (hunk 1 above)
Comment (Rodrive) [inline components/form/password.vue]: Same thing here : You have not disabled attrsInheritance so every attrs will be on `<div> `and `<input>` except style and class. For exemple you are calling this component with an id, this id will be on div and input

Comment (Nainterceptor) [reply components/form/password.vue]: Done !

**Pull request merged**